### ISSUE TITLE
Component: Dropdown | dropdown box missing the required children for the given role.

### ISSUE THREAD

**Issue by WTW-Jacob-Evans** [primefaces/primeng#15123]
### Describe the bug

using Axe DevTools, the dropdown component, when empty, shows an accessibility issue relating to the dropdown box missing the required children for the given role.
![image](https://github.com/primefaces/primeng/assets/163871134/3e8701aa-1a56-426f-b026-a532c74c5a3e)
![image](https://github.com/primefaces/primeng/assets/163871134/05d0f466-f957-4f01-81f3-5764f495941e)

This is present in the attached stackblitz sample, using the Axe DevTools.
![image](https://github.com/primefaces/primeng/assets/163871134/d7c0abb2-5641-4d69-8da5-e8a472842b10)

### Environment

Windows 10

### Reproducer

https://stackblitz.com/edit/github-2ten9s?file=src%2Fapp%2Fapp.component.html,src%2Fapp%2Fapp.component.ts,src%2Fapp%2Fapp.module.ts

### Angular version

17.2.3

### PrimeNG version

17.11.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0
